eventsources/sources/gitlab/start.go

```diff
@@ -86,8 +86,8 @@ func (router *Router) HandleRoute(writer http.ResponseWriter, request *http.Requ
 			return
 		}
 	}
-
-	body, err := ioutil.ReadAll(request.Body)
+	request.Body = http.MaxBytesReader(writer, request.Body, 65536)
```

Comment:
@whynowy sorry for the delay, I am just now seeing this - Does that mean we are no longer supporting gitlab payloads above 65KB? (same question goes for bitbucket, bitbucket server and webhook eventsources)

Reply:
@whynowy and btw do we have a different behavior atm in github? I wasn't able to find the same change in github eventsource

Reply:
From my experience, webhook payloads can grow up to 1MB and even more in some cases, especially with monorepos. I think the best solution would be to allow the users configure the max payload size by themselves (and set some default value in case they don't). I suggest to add an optional `MaxPayloadSize` field under `WebhookContext` which is used by all of the git/webhook related eventsources, wdyt @whynowy ?

Reply:
This is a good idea @daniel-codefresh .

Reply:
I've created a new issue on the subject and assigned to myself: #2088

Reply:
Much appreciated!